### PR TITLE
Hotfix for 0.30.1

### PR DESCRIPTION
**Pull request by akaszynski**
Add a hotfix for the scalar bar attribute and a quick ravel for center on ``CircularArc``.

### CODE REVIEW

File: pyvista/utilities/geometric_objects.py [hunk 1]
```diff
@@ -661,7 +661,7 @@ def CircularArc(pointa, pointb, center, resolution=100, negative=False):
     angle = np.deg2rad(arc.GetAngle())
     arc = pyvista.wrap(arc.GetOutput())
     # Compute distance of every point along circular arc
-    center = np.array(center)
+    center = np.array(center).ravel()
```
Comment (tkoyama010) [inline pyvista/utilities/geometric_objects.py:664]: @akaszynski Thank you for fix. Could you explain why this change is need?

Comment (adeak) [reply pyvista/utilities/geometric_objects.py:664]: Good question, this seems superfluous for a flat 3-length `center`.

**Review by GuillaumeFavelier — COMMENTED** [review]
Also I think updating a little test to cover the new `scalar_bar` property would be good to have.

File: pyvista/utilities/geometric_objects.py (hunk 1 above)
Comment (akaszynski) [reply pyvista/utilities/geometric_objects.py:664]: If for whatever reason someone puts in a center that's ``[[1, 2, 3]]``

Comment (akaszynski) [reply pyvista/utilities/geometric_objects.py:664]: Reason why I added this was I'm getting a failure in one of my repositories and figured that it's better to simply fix this here since others might encounter the same issue.

Comment (adeak) [reply pyvista/utilities/geometric_objects.py:664]: Slightly unusual approach :laughing: Harmless, I guess.
